routes.js

```diff
@@ -15,5 +15,27 @@ module.exports = (app) => {
                 res.status(404).json({error: 'Not found'})
             }
         })
-    })
-};
\ No newline at end of file
+    });
+
+    app.get('/profile', isLoggedIn, (req, res) => {
+        res.json(req.user);
+    });
+
+    // Facebook authentication
+    app.get('/auth/fb', passport.authenticate('facebook', {scope: ['email']}));
+    app.get('/auth/fb/callback', (req, res, next) => {
+        return passport.authenticate('facebook', {
+            successRedirect: req.session.returnTo || '/profile',
+            failureRedirect: '/'
+        })(req, res, next);
+    });
+};
+
+function isLoggedIn(req, res, next) {
+
```

Comment:
Naming is confusing:
_isLoggedIn_  suggests function is pure getter returning boolean 

But it is middleware function with redirects.
Propose chaneg to smth like _'authMiddleware'_  _'loggedInMiddleware'_